addon/components/paper-item.js

```diff
@@ -2,6 +2,19 @@ import Ember from 'ember';
 import RippleMixin from '../mixins/ripple-mixin';
 import ProxyMixin from 'ember-paper/mixins/proxy-mixin';
 
+const {
+  get,
+  set,
+  isEmpty,
+  computed,
+  run: { later }
```

Comment:
We're not destructuring more than one level. It helps to still have some context.
So, we should `run.later` instead of `later`.